### PR TITLE
QA Observation bugfix/FOUR-13196: Behaviors for My Requests My Task tables

### PR DESCRIPTION
**Pull request by CarliPinell**
## How to Test
- Create Requests and Tasks with Admin User and Participant User without permissions
- Asign Start permission to a participant user in some process
- Start Request With both type of users

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-13196

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.
ci:deploy
ci:next

### CODE REVIEW

File: resources/js/processes-catalogue/components/RequestTab.vue
Comment (fagubla) [inline resources/js/processes-catalogue/components/RequestTab.vue:1]: pmqlRequest is not used, remove it

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-a2f255dc4a.engk8s.processmaker.net

**Comment by pmPaulis** [comment]
@CarliPinell 
I reviewed in the TASK in the build 
this is the expected behaviur 
image
but only show this
image

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-a2f255dc4a.engk8s.processmaker.net

File: resources/js/processes-catalogue/components/RequestTab.vue
Comment (CarliPinell) [reply resources/js/processes-catalogue/components/RequestTab.vue:1]: solved

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-a2f255dc4a.engk8s.processmaker.net

File: resources/js/processes-catalogue/components/RequestTab.vue [hunk 1]
```diff
@@ -168,8 +167,10 @@ export default {
     },
     queryBuilder() {
       let pmql = "";
-      if (this.pmqlRequest !== undefined) {
-        pmql = this.pmqlRequest;
+      if (this.pmqlRequest !== undefined && this.pmqlRequest !== "") {
+        pmql = this.pmqlRequest + " AND process_id=" + `${this.process.id}`;
+      } else {
+        pmql = " process_id=" + `${this.process.id}`;
```
Comment (fagubla) [inline resources/js/processes-catalogue/components/RequestTab.vue:173]: `pmqlRequest` does not exist, there will be an error

**Review by fagubla — CHANGES_REQUESTED** [review]

File: resources/js/processes-catalogue/components/RequestTab.vue (hunk 1 above)
Comment (CarliPinell) [reply resources/js/processes-catalogue/components/RequestTab.vue]: Solved. Only pmql variable is used. The If ELSE code was removed

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-a2f255dc4a.engk8s.processmaker.net

File: resources/js/processes-catalogue/components/RequestTab.vue (hunk 1 above)
Comment (CarliPinell) [reply resources/js/processes-catalogue/components/RequestTab.vue]: solved

**Pull request merged**